pandas/tests/extension/decimal/array.py

```diff
@@ -24,11 +26,14 @@ def construct_from_string(cls, string):
                             "'{}'".format(cls, string))
 
 
-class DecimalArray(ExtensionArray):
+class DecimalArray(ExtensionArray, ExtensionArithmeticMixin,
+                   ExtensionComparisonMixin):
     dtype = DecimalDtype()
 
     def __init__(self, values):
-        assert all(isinstance(v, decimal.Decimal) for v in values)
+        for val in values:
+            if not isinstance(val, self.dtype.type):
+                raise TypeError
```

Comment:
erorr message

Reply:
fixed (and also in pandas/tests/extension/json/array.py)